packages/fury-adapter-swagger/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Fury Swagger Parser Changelog
 
+## 0.28.3 (2020-02-17)
```

Comment:
```suggestion
## Master
```

As this isn’t released yet; we leave the heading to “Master” (representing the master branch) and gets the number assigned when we make a release.

Reply:
going to change